azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -619,32 +600,8 @@ public List<Integer> handle(final ResultSet rs) throws SQLException {
   public static class FetchExecutableFlows implements
       ResultSetHandler<List<ExecutableFlow>> {
 
-    static String FETCH_EXECUTABLE_FLOW_BY_START_TIME =
-        "SELECT ef.exec_id, ef.enc_type, ef.flow_data, ef.status FROM execution_flows ef WHERE "
-            + "project_id=? AND flow_id=? AND start_time >= ? ORDER BY start_time DESC";
-    static String FETCH_BASE_EXECUTABLE_FLOW_QUERY =
-        "SELECT ef.exec_id, ef.enc_type, ef.flow_data, ef.status FROM execution_flows ef";
-    private static final String FETCH_FLOWS_STARTED_BEFORE = FETCH_BASE_EXECUTABLE_FLOW_QUERY +
-        " WHERE start_time < ?";
-    static String FETCH_EXECUTABLE_FLOW =
-        "SELECT exec_id, enc_type, flow_data, status FROM execution_flows "
-            + "WHERE exec_id=?";
-    static String FETCH_ALL_EXECUTABLE_FLOW_HISTORY =
-        "SELECT exec_id, enc_type, flow_data, status FROM execution_flows "
-            + "ORDER BY exec_id DESC LIMIT ?, ?";
-    static String FETCH_EXECUTABLE_FLOW_HISTORY =
-        "SELECT exec_id, enc_type, flow_data, status FROM execution_flows "
-            + "WHERE project_id=? AND flow_id=? "
-            + "ORDER BY exec_id DESC LIMIT ?, ?";
-    static String FETCH_EXECUTABLE_FLOW_BY_STATUS =
-        "SELECT exec_id, enc_type, flow_data, status FROM execution_flows "
-            + "WHERE project_id=? AND flow_id=? AND status=? "
-            + "ORDER BY exec_id DESC LIMIT ?, ?";
-    // Fetch flows that are in preparing state for more than a certain duration.
-    private static final String FETCH_FLOWS_QUEUED_FOR_LONG_TIME =
-        "SELECT exec_id, enc_type, flow_data, status FROM execution_flows"
-            + " WHERE submit_time < ? AND status = "
-            + Status.PREPARING.getNumVal();
+    static String FETCH_EXECUTABLE_FLOW_BASE_QUERY =
+        "SELECT ef.exec_id, ef.enc_type, ef.flow_data, ef.status FROM execution_flows ef ";
 
```

Comment:
Can we manage to keep the corresponding new constant the same place / order so maybe github can help display it in a way that's easier to read the diff?

Reply:
Agree. Let me do it

Reply:
Fixed